lib/resty/apisix/upstream.lua

```diff
@@ -1,18 +1,20 @@
 local ffi = require("ffi")
 local base = require("resty.core.base")
 local get_request = base.get_request
+local get_phase = ngx.get_phase
 local C = ffi.C
 local NGX_ERROR = ngx.ERROR
+local NGX_OK = ngx.OK
 
 
 base.allows_subsystem("http")
 
 
-ffi.cdef([[
+ffi.cdef[[
```

Comment:
I prefer the old style, pls keep the old style

Reply:
Done.